index.html

```diff
@@ -0,0 +1,390 @@
+                        distinctio consectetur voluptate magnam esse atque quia cumque sequi</p>
+                </div>
+                <div class="team__members">
+                    <div class="team__member">
+                        <img class="team__member__img" src="img/member__1.png"/>
+                        <p class="team__member__name">John Doe</p>
+                        <p class="team__member__position">General Manager</p>
+                        <div class="team__member__social">
+                            <a class="team__member__link" href="#"><img src="img/facebook.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/twitter.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/comments.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/dribbble.png" class="team__member__button"></a>
+                        </div>
+                    </div>
+                    <div class="team__member">
+                        <img class="team__member__img" src="img/member__2.png"/>
+                        <p class="team__member__name">Alexa Biru</p>
+                        <p class="team__member__position">Office Director</p>
+                        <div class="team__member__social">
+                            <a class="team__member__link" href="#"><img src="img/facebook.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/twitter.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/comments.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/dribbble.png" class="team__member__button"></a>
+                        </div>
+                    </div>
+                    <div class="team__member">
+                        <img class="team__member__img" src="img/member__3.png"/>
+                        <p class="team__member__name">Ronobir sing</p>
+                        <p class="team__member__position">Admin</p>
+                        <div class="team__member__social">
+                            <a class="team__member__link" href="#"><img src="img/facebook.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/twitter.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/comments.png" class="team__member__button"></a>
+                            <a class="team__member__link" href="#"><img src="img/dribbble.png" class="team__member__button"></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="blog">
```

Comment:
1) Reduce buttons sizes for mobiles/tablets
2) Remove the ability to change size for <textarea> block
3) Reduce font-sizes for the Blog block